src/AppBundle/DataFixtures/ORM/LoadAdherentData.php

```diff
@@ -112,20 +114,36 @@ public function load(ObjectManager $manager)
             'registered_at' => '2017-01-16 18:33:22',
         ]);
 
+        $adherent7 = $adherentFactory->createFromArray([
+            'uuid' => self::ADHERENT_7_UUID,
+            'password' => 'Champion20',
+            'email' => 'francis.brioul@yahoo.com',
+            'gender' => 'male',
+            'first_name' => 'Francis',
+            'last_name' => 'Brioul',
+            'address' => PostAddress::createFrenchAddress('39 rue de Melun', '77190-77152'),
+            'birthdate' => '1962-01-07',
+            'position' => 'employed',
+            'phone' => '33 673654349',
+            'registered_at' => '2017-01-25 19:31:45',
+        ]);
+
         // Create adherents accounts activation keys
         $key1 = AdherentActivationToken::generate($adherent1);
         $key2 = AdherentActivationToken::generate($adherent2);
         $key3 = AdherentActivationToken::generate($adherent3);
         $key4 = AdherentActivationToken::generate($adherent4);
         $key5 = AdherentActivationToken::generate($adherent5);
         $key6 = AdherentActivationToken::generate($adherent6);
+        $key7 = AdherentActivationToken::generate($adherent7);
 
         // Enable some adherents accounts
         $adherent2->activate($key2, '2016-11-16 20:54:13');
         $adherent3->activate($key3, '2017-01-03 09:12:37');
         $adherent4->activate($key4, '2017-01-18 13:23:50');
         $adherent5->activate($key5, '2017-01-08 06:42:56');
         $adherent6->activate($key6, '2017-01-17 08:07:45');
+        $adherent7->activate($key7, '2017-01-16 08:07:45');
```

Comment:
Your user registered on Jan. 25th but he activated his account on Jan. 16th. Not really valid ^^